packages/headless/src/features/product-recommendations/product-recommendations-actions.ts

```diff
@@ -201,32 +202,6 @@ const mapResultToProductResult = (
   };
 };
 
-const mapResultToDeprecatedProductResult = (
```

Comment:
@francoislg, fyi I went ahead and removed this function since properties like `sku` are no longer part of `ProductRecommendation` interface.

Reply:
All good